core-bundle/contao/languages/en/default.xlf

```diff
@@ -128,6 +128,9 @@
       <trans-unit id="ERR.invalidLogin">
         <source>Login failed (note that usernames and passwords are case-sensitive)!</source>
       </trans-unit>
+      <trans-unit id="ERR.insufficientAuthentication">
+        <source>Full authentication is required to access this resource.</source>
```

Comment:
The message is pretty technical and normal users might not know what "full authentication" is or what "this resource" is referring to. Can we think of a more helpful way of saying it?

Reply:
Message is now `Please re-authenticate to access this resource.`